aqua.yaml

```
@@ -10,8 +10,8 @@ registries:
 - type: standard
   ref: v4.190.0 # renovate: depName=aquaproj/aqua-registry
 packages:
-- name: kubernetes/kubectl@v1.29.4
-- name: kubernetes-sigs/kustomize@kustomize/v5.4.2
+- name: kubernetes/kubectl@v1.30.3
```

```suggestion
- name: kubernetes/kubectl@v1.30.4
```

How about using 1.30.4?

---

Thank you! I fixed it.